.eslintrc.cjs

```diff
@@ -10,7 +10,12 @@ const config = {
     serviceworker: true
   },
   globals: {
-    _: 'readonly'
+    _: 'readonly',
+    __webpack_public_path__: 'writeable',
```

Comment:
nice. i just addded one of these to my branch, but this is more expansive

Reply:
apparently `writeable` vs `writable` is a big controversy:
https://github.com/eslint/eslint/pull/11338#discussion_r252495166